src/modules/commander/HealthAndArmingChecks/checks/sdcardCheck.cpp

```diff
@@ -38,12 +38,14 @@
 #ifdef __PX4_DARWIN
 #include <sys/param.h>
 #include <sys/mount.h>
-#else
+#elif defined(PX4_STORAGEDIR)
```

Comment:
I'm pretty sure all of these platform defines could be cleaned up and removed, but this is good enough for now.

I suppose ideally we'd be able to opt out of this higher level and not have sdcardCheck included at all.